core/trino-main/src/test/java/io/trino/execution/buffer/BenchmarkBlockSerde.java

```diff
@@ -27,6 +27,7 @@
 import io.trino.spi.type.DecimalType;
```

Comment:
merge `Add serializeRow and deserializeRow to BenchmarkBlockSerde` and ` Add support for RowType to BenchmarkDataGenerator` commits together